bin/build-plugin-zip.sh

```diff
@@ -1,21 +1,31 @@
 #!/bin/bash
 
-# Exit if any command fails
+# Exit if any command fails.
 set -e
 
-# Change to the expected directory
+# Change to the expected directory.
 cd "$(dirname "$0")"
 cd ..
 
-# Enable nicer messaging for build status
+# Enable nicer messaging for build status.
+GREEN_BOLD='\033[1;32m';
+RED_BOLD='\033[1;31m';
 YELLOW_BOLD='\033[1;33m';
 COLOR_RESET='\033[0m';
 status () {
+	echo -e "\n${GREEN_BOLD}$1${COLOR_RESET}\n"
```

Comment:
Should `status` use green, or something success-neutral?

Reply:
Yeah maybe not green actually, good point. I was trying to say "this is okay, it's just informational" but green is too happy. I'll save that for the "Done" message.